dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -48,12 +50,28 @@
 
 import com.google.cloud.tools.opensource.dependencies.Artifacts;
 import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
+import com.google.common.collect.ImmutableList;
 import com.google.common.io.MoreFiles;
 import com.google.common.io.RecursiveDeleteOption;
+import com.google.common.truth.Correspondence;
 import com.google.common.truth.Truth;
 
 public class DashboardTest {
 
+  private static final Correspondence<Node, String> NODE_VALUES =
+      new Correspondence<Node, String>() {
+        @Override
+        public boolean compare(Node node, String expected) {
+          String nodeValue = node.getValue().trim();
+          return nodeValue.equals(expected);
+        }
+
+        @Override
+        public String toString() {
+          return "has value equal to";
```

Comment:
This is strange. Why do we need this method? and returning a constant?

Reply:
This is to help Correspondence's error message:

> suitable to fill the gap in a failure message of the form "<some actual element> is an element that ... <some expected element>"
https://google.github.io/truth/api/0.40/com/google/common/truth/Correspondence.html#toString--